src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,83 @@
+import { FC, useState } from 'react';
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+interface Props {
+  people: Person[];
+}
+
+export const PeopleTable: FC<Props> = ({ people }) => {
+  const [selectedPerson, setSelectedPerson] = useState('');
```

Comment:
![image](https://user-images.githubusercontent.com/106398574/186177430-fa689a7c-cb1d-4225-b0ad-4fc067d3a435.png)
you need useParams here